### ISSUE TITLE
Wrong cfn template parameter for Cognito UserPoolID in function template

### ISSUE THREAD

**Issue by JIoJIaJIu** [aws-amplify/amplify-cli#1611]
**Describe the bug**
When the function is created with `amplify-cli` and permissions for congito are provided the cfn template contains wrong parameter name

**To Reproduce**
1. `amplify function add` with congito permissions
2. `amplify push` outputs
```
CREATE_FAILED               functionmylambda         AWS::CloudFormation::Stack Fri Jun 07 2019 15:13:20 GMT+0800 (Central Indonesia Time) Template format error: Par
ameter name AUTH_COGNITOXXXXX_USERPOOLID is non alphanumeric.
```
3. Template `mylambda-cloudformation-template.json` contains
```
  ...
    "Parameters": {
        "env": {
            "Type": "String"
        },
        "AUTH_COGNITOXXXXXXX_USERPOOLID": {
            "Type": "String",
            "Default": "AUTH_COGNITOXXXXXXX_USERPOOLID"
        }
    },
  ...
```

**Expected behavior**
Template `mylambda-cloudformation-template.json` should contain
```
    "Parameters": {
        "env": {
            "Type": "String"
        },
        "authcognitoxxxxxxUserPoolId": {
            "Type": "String",
            "Default": "AUTH_COGNITOXXXXXXX_USERPOOLID"
        }
    },
```

**Desktop (please complete the following information):**
 - ```
     $ (master) amplify --version
    1.7.0
    ```

BTW, I can provide PR for it if needs

**Comment by kaustavghosh06** [comment]
@JIoJIaJIu We’ve fixed this in 1.7.1 version.

**Comment by JIoJIaJIu** [comment]
@kaustavghosh06 thank you
sorry, I haven't seen similar issue
